samples/unit-tests/stock-tools/stock-annotation-dragging/demo.js

```diff
@@ -0,0 +1,72 @@
+QUnit.test('Dragging annotation', function (assert) {
+    const data = [];
+    for (let i = 0; i < 300; i++) {
+        if (i > 100 && i < 200) {
+            continue;
+        }
```

Comment:
Would it perhaps make sense to invert the if statement in order to avoid continue?

Reply:
Doesn't matter that much, apart from that I usually prefer early return, which this continue kinda is.